public/app/features/provisioning/api/endpoints.ts

```diff
@@ -167,6 +167,12 @@ const injectedRtkApi = api
         }),
         providesTags: ['Repository'],
       }),
+      getRepositoryList: build.query<GetRepositoryListResponse, GetRepositoryListArg>({
```

Comment:
How's this different from the `/repositories` endpoint?

Reply:
oh interesting -- this is listing within a repository; not the list of repositories 🤔 

EG:

http://localhost:3000/apis/provisioning.grafana.app/v0alpha1/namespaces/default/repositories/github-ryan/list

shows the list of resources saved in grafana that belong to `github-ryan`

Reply:
So the same as a file list endpoint? I think the tags might need to be updated, because now it uses the same tag as repository list.